src/components/Expander/Expander.less

```diff
@@ -44,7 +44,7 @@
 	}
 
 	&-text {
-		.transition-group-animation-fade(100ms);
+		.transition-group-animation-fade(@timing-animation-hover);
```

Comment:
I also moved some of the animation timings into a variable as part of the cleanup. No functional changes.